plugins/javaembed/HpccClassLoader.java

```diff
@@ -71,16 +71,17 @@ private HpccClassLoader(String classPath, ClassLoader parent, int _bytecodeLen,
     }
     public synchronized Class<?> findClass(String className) throws ClassNotFoundException
     {
-        Class<?> result = classes.get(className);
```

Comment:
This was causing internal errors when passing datasets using iterators, and represents a 7.2.x regression